AdyenActions/Actions/OXXOVoucherAction.swift

```diff
@@ -18,20 +19,31 @@ public class OXXOVoucherAction: GenericVoucherAction, DownloadableVoucher {
     
     /// Download URL.
     public let downloadUrl: URL
+    
+    /// The instruction url.
+    @available(*, deprecated, message: "Please use `instructionsURL` instead.")
```

Comment:
So we basically hoping to "cover up" breaking change with this deprecated line, right?